examples/navier-stokes/Makefile

```diff
@@ -14,7 +14,7 @@
 # software, applications, hardware, advanced system engineering and early
 # testbed platforms, in support of the nation's exascale computing imperative.
 
-OPT ?= -O -g -march=native -ffp-contract=fast -fopenmp-simd
+OPT ?= -O3 -g -march=native -ffp-contract=fast -fopenmp-simd
```

Comment:
Is this intentional? (I don't object, but it's unclear whether you wanted to add it.)